packages/ember-metal/lib/path_cache.js

```diff
@@ -4,10 +4,10 @@ var IS_GLOBAL      = /^([A-Z$]|([0-9][A-Z$]))/;
 var IS_GLOBAL_PATH = /^([A-Z$]|([0-9][A-Z$])).*[\.]/;
 var HAS_THIS       = 'this.';
 
-var isGlobalCache       = new Cache(1000, function(key) { return IS_GLOBAL.test(key);          });
-var isGlobalPathCache   = new Cache(1000, function(key) { return IS_GLOBAL_PATH.test(key);     });
-var hasThisCache        = new Cache(1000, function(key) { return key.lastIndexOf(HAS_THIS, 0) === 0; });
-var firstDotIndexCache  = new Cache(1000, function(key) { return key.indexOf('.');             });
+var isGlobalCache       = new Cache(1000, (key) => { IS_GLOBAL.test(key); });
+var isGlobalPathCache   = new Cache(1000, (key) => { IS_GLOBAL_PATH.test(key); });
+var hasThisCache        = new Cache(1000, (key) => { return key.lastIndexOf(HAS_THIS, 0) === 0; });
+var firstDotIndexCache  = new Cache(1000, (key) => { key.indexOf('.'); });
```

Comment:
The justification for multiple lines is the same as elsewhere: We should be able to drop a debugger in there easily. Multiple lines please.